libs/experimental/langchain_experimental/llms/ollama_functions.py

```diff
@@ -136,6 +137,9 @@ def parse_response(message: BaseMessage) -> str:
     raise ValueError(f"`message` is not an instance of `AIMessage`: {message}")
 
 
+@deprecated(  # type: ignore[arg-type]
+    since="0.0.64", removal="0.3.0", alternative_import="langchain_ollama.ChatOllama"
```

Comment:
Let's set this to 0.4.0 or later? (I think for 0.3.0 we are considering just making pydantic changes).